tests/testthat/test_ari_narrate.R

```diff
@@ -16,7 +21,7 @@ skip_narrate <- function(){
 
 video <- file.path(tempdir(), "output.mp4")
 #video <- file.path(getwd(), "output.mp4")
-if (!nzchar(Sys.getenv("AWS_ACCESS_KEY_ID"))) {
+if (nzchar(Sys.getenv("AWS_ACCESS_KEY_ID"))) {
```

Comment:
I wonder why I had this the original way? 🤔

Reply:
Don't know - I think it didn't matter because you had valid credentials on Travis.  CRAN won't run for any instances anyway.